test/abort/test-addon-uv-handle-leak.js

```diff
@@ -116,8 +119,12 @@ if (process.argv[2] === 'child') {
         }
         state = 'handle-start';
         break;
+      case 'source-line':
+        assert.match(line, /CheckedUvLoopClose/);
```

Comment:
@joyeecheung should this check be scoped only to linux ? Running this test on windows fails with

```
node:assert:1035
    throw err;
    ^

AssertionError [ERR_ASSERTION]: The input did not match the regular expression /CheckedUvLoopClose/. Input:

'#  Windows PowerShell[6632]:  at C:\\Users\\demohan\\github\\node\\src\\debug_utils.cc:344'

    at Object.<anonymous> (C:\Users\demohan\github\node\test\abort\test-addon-uv-handle-leak.js:125:16)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Module._load (node:internal/modules/cjs/loader:1023:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  generatedMessage: true,
  code: 'ERR_ASSERTION',
  actual: '#  Windows PowerShell[6632]:  at C:\\Users\\demohan\\github\\node\\src\\debug_utils.cc:344',
  expected: /CheckedUvLoopClose/,
  operator: 'match'
}
```
which is expected given lack of function name in https://github.com/nodejs/node/blob/8382da1bc1dc4b2a4fb70ae538611477a5fea31a/src/util.h#L165

Are the addon tests not run on windows CI ?

Reply:
I believe the abort tests are not run in the Windows CI because our test runner is written in python and it uses `resource.RLIMIT_CORE` to disable core file, which isn't available on Windows.

Reply:
From a look of the docs, I am guessing we can use `__FUNCSIG__` as a replacement on Windows https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170

Reply:
Thanks for the clarification!